<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="book_info_layout" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#book_info_layout{
border-style: solid;
border-color: rgb(234, 234, 234);
border-width: 5px;
border-radius: 10px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,0,5,0">
      <item row="3" column="2">
       <widget class="QLineEdit" name="author">
        <property name="maximumSize">
         <size>
          <width>520</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="price"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Количество</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPlainTextEdit" name="description">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>520</width>
          <height>250</height>
         </size>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="add_to_cart_button">
        <property name="text">
         <string>Добавить в корзину</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="number_sold">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="quantity"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Автор</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="info">
        <property name="styleSheet">
         <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="sold">
        <property name="text">
         <string>Всего продано:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="title"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="delete_button">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Описание</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="quantity_spin"/>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="save_changes">
        <property name="text">
         <string>Сохранить изменения</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="finish_add_new_book">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
